glean-core/ios/Glean/Metrics/DistributionData.swift

```diff
@@ -14,6 +14,9 @@ public class DistributionData {
     let values: [UInt64: UInt64]
     let sum: UInt64
 
+    /// Parse the distribution data from the given JSON string.
+    ///
+    /// If the string is not valid JSON or data is missing `nil` is returned.
```

Comment:
I'm not sure it's entirely necessary for all functions but you omitted the `- parameters` section for this function.

Reply:
yeah I was like "do I really need to document a very well named parameter?"

Reply:
Good enough explanation for me :))